examples/gatsbygram/src/components/Avatar.js

```diff
@@ -0,0 +1,49 @@
+import * as PropTypes from "prop-types"
+import React from "react"
+import { rhythm } from "../utils/typography"
+import { graphql } from "gatsby"
+
+const propTypes = {
+  user: PropTypes.shape({
+    avatar: PropTypes.string.isRequired,
+    username: PropTypes.string.isRequired,
+  }),
+}
+
+function Avatar({ user }) {
+  return (
+    <img
+      src={user.avatar}
+      alt={user.username}
+      css={{
+        display: `block`,
+        margin: `0 auto`,
+        borderRadius: `100%`,
+        width: rhythm(2),
+        height: rhythm(2),
+        [`@media (min-width: 460px)`]: {
+          width: rhythm(3),
+          height: rhythm(3),
+        },
+        [`@media (min-width: 525px)`]: {
+          width: rhythm(4),
+          height: rhythm(4),
+        },
+        [`@media (min-width: 600px)`]: {
+          width: rhythm(6),
```

Comment:
we've used the value `inherit` before, what's the reason for using `rhythm(6)`?

Reply:
As the avatar for some reason is very low resolution, using 'inherit' doesn't look right, instead I chose 'rhythm(6)'.

I can change it back if you like it better the other way.

Reply:
Yeah, we could just remove it - it's simpler that way

Reply:
@wardpeet Just tested it - The image is way to small without the media query on my screen.

Reply:
mind showing a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/1561200/58032764-98d9e780-7b23-11e9-91c2-62a3e6ef931d.png)

![image](https://user-images.githubusercontent.com/1561200/58032794-a2634f80-7b23-11e9-9663-e32986f19e9b.png)

First one is with my changes, last one is without media query

Reply:
oh weird, well just go with yours, it's no big deal anyway 😄